tests/bcpLanguageTagChecker.js

```diff
@@ -50,4 +52,14 @@ describe('bcpLanguageTagChecker', function () {
       expect(bcpLanguageTagChecker(incorrectLanguageTag)).to.be.false
     })
   })
+
+  describe('isPrivateLanguage()', function () {
+    const privateUseLanguages = ['qtx']
```

Comment:
Maybe also add some in the range here?
```suggestion
    const privateUseLanguages = ['qaa', 'qtx', 'qtz']